azure/functions/decorators/function_app.py

```diff
@@ -1435,6 +1443,56 @@ def decorator():
 
         return wrap
 
+    def assistant_skill_trigger(self,
+                                arg_name: str,
+                                task_description: str,
+                                data_type: Optional[
```

Comment:
AssistantSkillTrigger also supports supplying three more optional parameters - 
![image](https://github.com/Azure/azure-functions-python-library/assets/67894494/cdec137a-178c-4e0c-972c-c79beec8e9ba)

Reply:
Added the remaining params